tests/tests_api/test_student.py

```diff
@@ -0,0 +1,91 @@
+from model_bakery import baker
+from rest_framework import status
+
+from api.models import AgeRange, DayAndTimeSlot, LanguageAndLevel, PersonalInfo, Student
+
+
+def test_student_create(api_client, faker):
+    initial_count = Student.objects.count()
+    personal_info = baker.make(PersonalInfo)
+    age_range_id = AgeRange.objects.first().id
+    teaching_languages_and_levels_ids = [
+        LanguageAndLevel.objects.first().id,
+        LanguageAndLevel.objects.last().id,
+    ]
+    availability_slots_ids = [
+        DayAndTimeSlot.objects.first().id,
+        DayAndTimeSlot.objects.last().id,
+    ]
+    m2m_fields = [
+        "teaching_languages_and_levels",
+        "availability_slots",
+    ]
```

Comment:
I've put it to variable to make it more clear - why we rewriting these fields filtering. Probably it could be a comment, but I think it's better with variable :)

Reply:
Moved it closer to loop